call-for-champions.md

```diff
@@ -0,0 +1,97 @@
+part of this.
+
+**Examples of good practice:** The publications from the Reproducible Research
+Champions will provide other researchers with good examples of what good
+reproducible research looks like.
+
+**Support for other researchers:** The programme will capture the experience
+and learning gained from making Champions' research reproducible to create a
+set of resources to support other researchers in doing the same. Being able to
+talk to researchers who have already gone through the process at the drop-in
+sessions will also help other researchers in making their own research
+reproducible.
+
+## What will be asked of me as a champion?
+**A paper:** You must have a paper you want to make reproducible that
+is either published or submitted for publication (pre-prints count).
+
+**Methods, data and code:** You must be happy for the detailed methods, data and
+code associated with your paper to be made available to other researchers, and
+ideally the public at large. We understand that not all data can be made available
+in a completely open manner for a range of reasons such as ethics, data protection,
+data ownership etc. Working out how to make research maximally reproducible in
+these scenarios is an important part of reducing the barriers to more widespread
+reproducibility, so if you are keen to be a Reproducible Research Champion but
+have restrictions on making data available, ask a question on the
+[#reproducible-research Slack channel](https://alan-turing-institute.slack.com/messages/C6XEYUQPR) or email [Kirstie Whitaker](mailto:kirstiej@turing.ac.uk) or [Martin O'Reilly](mailto:moreilly@turing.ac.uk).
+
+**Time:** You will need to provide some time to support:
+- The RSE working to ensure your paper is reproducible by answering questions about
+your methods, code or data, and helping to troubleshoot any issues they have
+reproducing your results. This will require a few hours a week for up to 4 weeks
+sometime during April, May or June. **[8-16 hours total]**
+- The capturing of learning from the programme as a set of resources to support
+other researchers to make their research reproducible. This will require your attendance at one or two review sessions after all Champions' papers have been
+made reproducible, as well as some time to review and provide feedback on the
+resources produced. **[8-12 hours total]**
+- Attending a subset of the reproducible research drop-in sessions to help other
+researchers in making their own research reproducible. **[6-10 hours total]**
+- The publicity campaign highlighting reproducible research best practice at the
+Turing. This will require writing a popular summary for your paper and
+contributing to publicity material such as blog posts, articles and interviews.
+**[2-4 days total]**
+
+## How do I apply?
+
+Applications should be sent to [Kirstie Whitaker](mailto:kirstiej@turing.ac.uk) by **Friday 23rd February**. These should
```

Comment:
Could make a little google form for this? Would be helpful from our end, and also more uniform I think...

Reply:
Agreed.